src/libbson/fuzz/fuzz_test_validate.c

```diff
@@ -0,0 +1,18 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <bson/bson.h>
+
+int
+LLVMFuzzerTestOneInput (const uint8_t *data, size_t size)
+{
+   bson_t b;
+   if (bson_init_static (&b, data, size)) {
+      bson_validate (&b,
+                     BSON_VALIDATE_UTF8 | BSON_VALIDATE_DOLLAR_KEYS | BSON_VALIDATE_DOT_KEYS |
+                        BSON_VALIDATE_UTF8_ALLOW_NULL | BSON_VALIDATE_EMPTY_KEYS,
+                     NULL);
+   }
+
+   return 0;
```

Comment:
```suggestion
      return 0;
   }
   return -1;
```
Do not add inputs which fail to initialize the bson_t object to the corpus. We are only interested in inputs which lead to `bson_validate()` being invoked.